### PR TITLE
Basic moderation backend

### PR DESCRIPTION
**Pull request by NunoDasNeves**
fixed some test stuff
moved backend test secrets out of repo
superuser now exists, can delete and edit arbitrary comments/questions/reviews

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "start": "cross-env NODE_ENV=test LOG=true npx nodemon ./src/bin/www",
     "lint": "npx eslint src/**/*.js tests/**/*.js --fix",
     "test:lint": "npx eslint src/**/*.js tests/**/*.js",
-    "test:src": "cross-env NODE_ENV=test npx nyc mocha tests/**/*.js --timeout 75000",
+    "test:src": "cross-env NODE_ENV=test npx nyc mocha --file tests/utils/*.js --file tests/routes/*.js --timeout 75000",
```
Comment (DarkPurple141) [inline backend/package.json]: This timeout is probably excessive now.

File: backend/src/models/constants.js [hunk 2]
```diff
@@ -16,6 +16,12 @@ exports.MAX_ENJOY = 5
 exports.MIN_OPTION = 0
 exports.MAX_OPTION = 3
 
+// Permissions Constants (TODO:this properly, more granularity etc)
```
Comment (DarkPurple141) [inline backend/src/models/constants.js]: Although verbose; I think `PERMISSIONS_*` is better.

File: backend/src/models/constants.js [hunk 3]
```diff
@@ -356,6 +362,10 @@ exports.TABLE_COLUMNS = {
         picture: {
             type: TYPES.VarChar,
             options: { nullable: true }
+        },
+        permissions: {
```
Comment (DarkPurple141) [inline backend/src/models/constants.js]: This should probably be a table? To enforce `enum`-ness.

Comment (NunoDasNeves) [reply backend/src/models/constants.js]: Yeah maybe but then joinszzz hnng. We have constants - are they not good enough?
We will need another table for a more complex permissions model but until then...cbf?

Comment (DarkPurple141) [reply backend/src/models/constants.js]: okeyyy

**Pull request merged**